commit 27582fa9342d5d677c6588c3cc8b1f8821cd69df
Author: Takashi Iwai <tiwai@suse.de>
Date:   Tue Sep 4 17:58:46 2018 +0200

    staging: bcm2835-audio: Add 10ms period constraint
    
    commit 93c66acaf68b5247c3121a46a71ff6a70fc1d492 upstream.
    
    It seems that the resolution of vc04 callback is in 10 msec; i.e. the
    minimal period size is also 10 msec.
    
    This patch adds the corresponding hw constraint.
    
    Signed-off-by: Takashi Iwai <tiwai@suse.de>
    Tested-by: Stefan Wahren <stefan.wahren@i2se.com>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>

diff --git a/drivers/staging/vc04_services/bcm2835-audio/bcm2835-pcm.c b/drivers/staging/vc04_services/bcm2835-audio/bcm2835-pcm.c
index 9659c25b9f9d..6d89db6e14e4 100644
--- a/drivers/staging/vc04_services/bcm2835-audio/bcm2835-pcm.c
+++ b/drivers/staging/vc04_services/bcm2835-audio/bcm2835-pcm.c
@@ -145,6 +145,11 @@ static int snd_bcm2835_playback_open_generic(
 				   SNDRV_PCM_HW_PARAM_PERIOD_BYTES,
 				   16);
 
+	/* position update is in 10ms order */
+	snd_pcm_hw_constraint_minmax(runtime,
+				     SNDRV_PCM_HW_PARAM_PERIOD_TIME,
+				     10 * 1000, UINT_MAX);
+
 	chip->alsa_stream[idx] = alsa_stream;
 
 	chip->opened |= (1 << idx);
diff -uN a/1.txt b/1.txt
--- a/dummy/rpi_1457_27582fa9342d5d677c6588c3cc8b1f8821cd69df.txt	1970-01-01 00:00:00.000000000 +0000
+++ b/dummy/rpi_1457_27582fa9342d5d677c6588c3cc8b1f8821cd69df.txt	2013-12-23 04:07:40.000000000 +0000
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
